deploy/mobile-client-crd.yaml

```diff
@@ -1,9 +1,9 @@
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
-  name: mobileclients.mobile.k8s.io
+  name: mobileclients.mobile.mdc.aerogear.org
```

Comment:
Looking at the rest of this PR, I think the `.mobile` part should also be removed here.

i.e.

```
mobileclients.mdc.aerogear.org
```
rather than

```
mobileclients.mobile.mdc.aerogear.org
```